articles/cognitive-services/Face/Tutorials/FaceAPIinJavaForAndroidTutorial.md

```diff
@@ -143,39 +148,45 @@ Follow the instructions below to configure the client library:
 		    }
 		}
 
-3. Open the build.gradle file in your 'app' project.
+3. Open the **build.gradle** file in your 'app' project.
 4. Add a dependency for our client library stored in the Maven Central Repository:
 
 		dependencies {  
 		    ...  
-		    compile 'com.microsoft.projectoxford:face:1.0.0'  
+		    compile 'com.microsoft.projectoxford:face:1.2.5'  
```

Comment:
Does this work on the latest version? The latest one should be `1.4.3` which is released just yesterday.